monero-outreach-docs/translations/nl/PSA/headers/PSA_nl_headers.md

```diff
@@ -0,0 +1,32 @@
+﻿# Headers
```

Comment:
# Headers

- Wallet 
   Ieders plicht:
   Monero uit het zicht

- Exchange
   zegt de Monero-held:
   kijk uit met je geld!

- Learning
   er zijn 52 leerweken in een jaar

- Don't talk
   zwijgen is goud

- Teamwork
   dat Monero team!
   nog nooit gezien

- Don't share
   mijn privésleutel met je delen?
   maar mijnheer, dat heet stelen

- Forks 
   vooruit met de vork